src/components/inline_edit/inline_edit_title.tsx

```diff
@@ -47,6 +49,10 @@ export const EuiInlineEditTitle: FunctionComponent<EuiInlineEditTitleProps> = ({
 }) => {
   const classes = classNames('euiInlineEditTitle', className);
 
+  const theme = useEuiTheme();
+  const styles = euiInlineEditTitleStyles(theme);
+  const cssStyles = styles.fontSize[size];
```

Comment:
Even though `euiInlineEditTitle` is empty, we still need the key here for Emotion to generate a readable/non-random class name
```suggestion
  const cssStyles = [styles.euiInlineEditTitle, styles.fontSize[size]];
```

Reply:
Gotcha! Good catch!